src/subprocess-posix.cc

```diff
@@ -169,11 +171,34 @@ void SubprocessSet::HandlePendingInterruption() {
     interrupted_ = SIGTERM;
 }
 
+void SubprocessSet::HandleChildDeath() {
```

Comment:
May be it's better to change the name to something more "politically correct". Or that's just me? ;)